### ISSUE TITLE
Sensory seeking

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7313]
**Preferred term label:**
Sensory seeking

**Synonyms**

**Definition (free text, please give PubMed ID)**
Unusual interest in sensory aspects of the environment.
PMID: 29112858
DSM-V

**Parent term (use hpo.jax.org/app)**
Stereotypy
HP:0000733

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
added as HP:4000079
